rex_xai/box.py

```diff
@@ -260,16 +442,27 @@ def add_children_to_tree(self, min_size, mode, map):
 
 
 def initialise_tree(
-    r_lim, c_lim, distribution, distribution_args, r_start=0, c_start=0
+    r_lim,
+    c_lim,
+    distribution,
+    distribution_args,
+    r_start=0,
+    c_start=0,
+    d_start=None,
+    d_lim=None,
 ) -> Box:
     """initialise box tree with root node, the whole image"""
+    if d_lim is not None:
+        d_start = 0
```

Comment:
```suggestion
    if d_lim is not None and d_start is None:
        d_start = 0
```

Currently, `d_start` is always set to zero if `d_lim` is provided. This change allows `d_start` to be set manually as an argument to `initialise_tree()`